eclair-core/src/main/scala/fr/acinq/eclair/NodeParams.scala

```diff
@@ -76,7 +77,7 @@ case class NodeParams(keyManager: KeyManager,
                       routerConf: RouterConf,
                       socksProxy_opt: Option[Socks5ProxyParams],
                       maxPaymentAttempts: Int) {
-
+  require(feeBase <= MilliSatoshi(0xFFFFFFFFL))
```

Comment:
This should be in `NodeParams.makeNodeParams` for consistency, with a human understandable message (and also a comment explaining why we have to cap the value)